Sources/PayPalMessages/PayPalMessageView.swift

```diff
@@ -235,11 +235,16 @@ public final class PayPalMessageView: UIControl {
 extension PayPalMessageView: PayPalMessageViewModelDelegate {
 
     /// Recreates the message content from the existing data. **Does not triggers a networking event.**
-    func refreshContent() {
+    func refreshContent(label: String, traits: UIAccessibilityTraits, accessibilityElement: Bool) {
         let params = viewModel.messageParameters
         messageLabel.attributedText = PayPalMessageAttributedStringBuilder().makeMessageString(params)
         // Force recalculation for layout
         invalidateIntrinsicContentSize()
+
+        // Update accessibility properties
+        self.accessibilityLabel = label
+        self.accessibilityTraits = traits
+        self.isAccessibilityElement = accessibilityElement
```

Comment:
`refreshContent()` now has multiple ways that it receives data from the view model:
1. Directly as method args
2. `viewModel.messageParameters`

Would it make sense to keep everything inside of the `PayPalMessageViewParameters` struct that we access via `viewModel.messageParameters` to keep the data flowing consistently?